Upstream-Status: Inappropriate [disable unused peripherals for faster boot]

diff --git a/arch/arm/boot/dts/ti/omap/am335x-bone-common.dtsi b/arch/arm/boot/dts/ti/omap/am335x-bone-common.dtsi
index 96451c8a815c..f924c5230e3f 100644
--- a/arch/arm/boot/dts/ti/omap/am335x-bone-common.dtsi
+++ b/arch/arm/boot/dts/ti/omap/am335x-bone-common.dtsi
@@ -209,7 +209,7 @@ &i2c0 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&i2c0_pins>;
 
-	status = "okay";
+	status = "disabled";
 	clock-frequency = <400000>;
 
 	tps: tps@24 {
@@ -233,7 +233,7 @@ &i2c2 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&i2c2_pins>;
 
-	status = "okay";
+	status = "disabled";
 	clock-frequency = <100000>;
 
 	cape_eeprom0: cape_eeprom0@54 {
@@ -359,6 +359,7 @@ ldo4_reg: regulator@6 {
 };
 
 &cpsw_port1 {
+	status = "disabled";
 	phy-handle = <&ethphy0>;
 	phy-mode = "mii";
 	ti,dual-emac-pvid = <1>;
@@ -376,6 +377,7 @@ &mac_sw {
 };
 
 &davinci_mdio_sw {
+	status = "disabled";
 	pinctrl-names = "default", "sleep";
 	pinctrl-0 = <&davinci_mdio_default>;
 	pinctrl-1 = <&davinci_mdio_sleep>;
@@ -398,11 +400,11 @@ &mmc1 {
 };
 
 &aes {
-	status = "okay";
+	status = "disabled";
 };
 
 &sham {
-	status = "okay";
+	status = "disabled";
 };
 
 &rtc {
@@ -412,7 +414,7 @@ &rtc {
 };
 
 &pruss_tm {
-	status = "okay";
+	status = "disabled";
 };
 
 &wkup_m3_ipc {
